client/post-editor/drafts-button/index.jsx

```diff
@@ -1,31 +1,43 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React, { PropTypes } from 'react';
 
 /**
  * Internal dependencies
  */
-var config = require( 'config' ),
-	Count = require( 'components/count' );
+import Count from 'components/count';
 
-var DraftsButton = React.createClass( {
+export default React.createClass( {
+	displayName: 'EditorDraftsButton',
 
-	render: function() {
-		if ( ! config.isEnabled( 'editor-drafts' ) || ! this.props.site ) {
+	propTypes: {
+		site: PropTypes.object,
+		count: PropTypes.number,
+		onClick: PropTypes.func
+	},
+
+	getDefaultProps() {
+		return {
+			count: 0,
+			onClick: () => {}
+		};
+	},
+
+	render() {
+		if ( ! this.props.site ) {
 			return null;
 		}
 
 		return (
-			<button className="drafts-button" onClick={ this.props.onClick } aria-label={ this.translate( 'View all drafts' ) }>
+			<button className="drafts-button"
+				onClick={ this.props.onClick }
+				disabled={ ! this.props.count }
+				aria-label={ this.translate( 'View all drafts' ) }
+			>
 				<span>{ this.translate( 'Drafts' ) }</span>
-				{ this.props.count ?
-					<Count count={ this.props.count } />
-				: null }
+				{ this.props.count && <Count count={ this.props.count } /> }
```

Comment:
I'm seeing a 0 at the end of the render.

![image](https://cloud.githubusercontent.com/assets/548849/11471169/975a7d22-9763-11e5-829b-cd27d95f4668.png)